rasa/nlu/tokenizers/tokenizer.py

```diff
@@ -102,6 +102,8 @@ def __init__(self, config: Dict[Text, Any]) -> None:
         self.token_pattern_regex = None
         if token_pattern:
             self.token_pattern_regex = re.compile(token_pattern)
+        # split intent to prefix and suffix greedily, None means don't split
+        self.prefix_separator_symbol = config.get("prefix_separator_symbol")
```

Comment:
I think this attribute should probably also be added to the default configs of the tokenizer subclasses